releases/release-1.31/release-notes/maps/pr-125265-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 125265
+releasenote:
+  text: 'kubeadm: during kubelet health checks, respect the healthz address:port configured
+    in the KubeletConfiguration instead of hardcoding localhost:10248.'
```

Comment:
```suggestion
  text: 'kubeadm: The healthz address:port configured in the KubeletConfiguration was used during kubelet health checks, instead of hardcoding localhost:10248.'
```